Modules/Loadable/VolumeRendering/MRMLDM/vtkMRMLVolumeRenderingDisplayableManager.cxx

```diff
@@ -165,6 +200,11 @@ vtkMRMLVolumeRenderingDisplayableManager::vtkInternal::vtkInternal(vtkMRMLVolume
 , OriginalDesiredUpdateRate(0.0) // 0 fps is a special value that means it hasn't been set
 , Interaction(0)
 {
+  this->MultiVolumeActor = vtkSmartPointer<vtkMultiVolume>::New();
+  this->MultiVolumeMapper = vtkSmartPointer<vtkGPUVolumeRayCastMapper>::New();
+  // Set GPU mapper to the multi-volume actor. Both objects are only used for GPU ray casting
```

Comment:
May be worth clarifying this comment

Reply:
I added a short description about how multi-volume is different from the others, and an ascii diagram (the same thing I had in front of me on a piece of paper while implementing, it's very helpful). I think it should be enough.